docs/en/01_usage.md

```diff
@@ -1,54 +1,68 @@
-# Using
+---
+title: TagField Module Usage
+summary: Managing tags on DataObjects, and integration with silverstripe-taxonomy
+icon: tags
+---
+
+# Usage
 
 The primary use, for this module, is as a custom input field interface. For instance, imagine you had the following data objects:
 
 ```php
+namespace App\Models;
+
 class BlogPost extends DataObject
```

Comment:
Please add appropriate `use` statements to this and all other PHP code examples.

Reply:
Updated